src/layouts/Layout.astro

```diff
@@ -102,6 +143,63 @@ const {
         text-align: center;
         padding: calc(var(--lh) * 2) var(--lh);
       }
+
```

Comment:
Same here, we should be only modifying local custom properties, without higher specificity CSS classes.